doc/rrdinfo.pod

```diff
@@ -5,7 +5,8 @@ rrdinfo - extract header information from an RRD
 =head1 SYNOPSIS
 
 B<rrdtool> B<info> I<filename>
-S<[B<--daemon>|B<-d> I<address> S<[B<--noflush>|B<-F>]>]>
+
+S<[B<--daemon>|B<-d> I<address> [B<--noflush>|B<-F>]>]>
```

Comment:
There is one '>' too much.
See: https://github.com/oetiker/rrdtool-1.x/blob/a9731c5243d73124711b8d43556a9ca0d6a6245d/doc/rrdinfo.pod
Should be:
`S<[B<--daemon>|B<-d> I<address> [B<--noflush>|B<-F>]]>`

Reply:
Yes, you're right. I thought I counted them. Strange podchecker didn't catch that. I can see how it got rendered in HTML that an extra `>` was present in the output. Thanks.